server/tests/participant-status.test.js

```diff
@@ -168,23 +168,23 @@ describe('Test Participant status data model and service', () => {
       city: 'Test City 1030',
     });
 
-    const emp1 = v4();
-    const emp2 = v4();
-    const emp3 = v4();
+    const emp1 = { id: v4(), sites: [site1.siteId, site2.siteId] };
```

Comment:
I think these tests make more sense. In the original, emp1 was prospecting the same participant for two sites, which I don't believe is a scenario.